src/settings/components/GeneralSettings.tsx

```diff
@@ -79,6 +79,13 @@ const GeneralSettings: React.FC<GeneralSettingsProps> = ({
         value={settings.defaultSaveFolder}
         onChange={(value) => updateSettings({ defaultSaveFolder: value })}
       />
+      <TextComponent
```

Comment:
This should be in QA settings instead.

Reply:
sure - I assume you'll move this around whenever the larger feature is out of QA?
also fixed a small typo I spotted in qa settings.